plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -780,14 +780,13 @@ private ConnectorTableMetadata getTableMetadata(ConnectorSession session, Schema
     private List<ColumnMetadata> getColumnMetadatas(Table table)
     {
         return table.schema().columns().stream()
-                .map(column -> {
-                    return ColumnMetadata.builder()
-                            .setName(column.name())
-                            .setType(toTrinoType(column.type(), typeManager))
-                            .setNullable(column.isOptional())
-                            .setComment(Optional.ofNullable(column.doc()))
-                            .build();
-                })
+                .map(column ->
```

Comment:
extract cleanups to a separate PR for quicker merge

Reply:
https://github.com/trinodb/trino/pull/11233